### DISCUSSION TITLE
Tutorial about deepcg

### DISCUSSION THREAD

**Discussion by katyo123** [deepmodeling/deepmd-kit#2942] [Q&A] [answered]
Hi！
I am trying to use deepcg, but the only available resource is this article: https://doi.org/10.1063/1.5027645 (I couldn't find any other information from Git or the article). Are there any other tutorials available?

**Comment by wanghan-iapcm** [comment]
The code used to train the deepcg model was deepmd-kit

**Reply by katyo123** [reply]
mean I need to modify the loss function , or other parts in kit to apply it to CG system?

**Reply by wanghan-iapcm** [reply]
no. The paper used the loc_frame descriptor.

**Comment by katyo123** [comment]
Dear Professor Wang
Thank you for your response! I noticed that the loss function used in deepcg is ![image](https://github.com/deepmodeling/deepmd-kit/assets/142722857/e977beff-d31d-4b90-887d-09677ab91837)
while in the kit, it is ![image](https://github.com/deepmodeling/deepmd-kit/assets/142722857/c85a0754-1730-4171-b588-7f79752cb4a0)
The loss function used for training is different.

**Reply by wanghan-iapcm** [reply, marked as answer]
you may set the energy and virial prefactors to 0 to obtain the deepcg loss.
